task_5.4.py

```diff
@@ -0,0 +1,22 @@
+def interleave(*args):
+    """
+    A function that accepts one or more iterable parameters,
+    and returns a list of the interwoven members
+    :param args: lists, tuples or strings
+    :return: a list of the interwoven members
+    """
+    for i in range(0, len(args[0])):
```

Comment:
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing